python3Packages.onecache: 0.7.1 -> 0.8.0

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/nix-community/nixpkgs-update) tools. This update was made based on information from passthru.updateScript.

meta.description for python3Packages.onecache is: Python LRU and TTL cache for sync and async code

meta.homepage for python3Packages.onecache is: https://github.com/sonic182/onecache

meta.changelog for python3Packages.onecache is: https://github.com/sonic182/onecache/blob/0.8.0/CHANGELOG.md


###### Updates performed
- Ran passthru.UpdateScript

###### To inspect upstream changes





###### Impact

<b>Checks done</b>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- found 0.8.0 with grep in /nix/store/pp1zd1sk5m8b7hm296fpb5inzndlikcl-python3.13-onecache-0.8.0
- found 0.8.0 in filename of file in /nix/store/pp1zd1sk5m8b7hm296fpb5inzndlikcl-python3.13-onecache-0.8.0

---

<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
5 total rebuild path(s)

5 package rebuild(s)

First fifty rebuilds by attrpath

python312Packages.aiosonic
python312Packages.onecache
python313Packages.aiosonic
python313Packages.onecache
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from the cache**:
```
nix-store -r /nix/store/pp1zd1sk5m8b7hm296fpb5inzndlikcl-python3.13-onecache-0.8.0 \
  --option binary-caches 'https://cache.nixos.org/ https://nixpkgs-update-cache.nix-community.org/' \
  --option trusted-public-keys '
  nixpkgs-update-cache.nix-community.org-1:U8d6wiQecHUPJFSqHN9GSSmNkmdiFW7GW7WNAnHW0SM=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The nixpkgs-update cache is only trusted for this store-path realization.)
For the cached download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A python3Packages.onecache https://github.com/r-ryantm/nixpkgs/archive/18bb533e337d74f2b29167fd5fa949c836b2701a.tar.gz
```
Or:
```
nix build github:r-ryantm/nixpkgs/18bb533e337d74f2b29167fd5fa949c836b2701a#python3Packages.onecache
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/pp1zd1sk5m8b7hm296fpb5inzndlikcl-python3.13-onecache-0.8.0
ls -la /nix/store/pp1zd1sk5m8b7hm296fpb5inzndlikcl-python3.13-onecache-0.8.0/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review --extra-nixpkgs-config '{ allowBroken = false; }'`
Commit: `18bb533e337d74f2b29167fd5fa949c836b2701a`

---
### `x86_64-linux`
<details>
  <summary>:x: 4 packages failed to build:</summary>
  <ul>
    <li>python312Packages.aiosonic</li>
    <li>python312Packages.aiosonic.dist</li>
    <li>python313Packages.aiosonic</li>
    <li>python313Packages.aiosonic.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 4 packages built:</summary>
  <ul>
    <li>python312Packages.onecache</li>
    <li>python312Packages.onecache.dist</li>
    <li>python313Packages.onecache</li>
    <li>python313Packages.onecache.dist</li>
  </ul>
</details>

---

###### Maintainer pings

cc @geraldog for [testing](https://github.com/nix-community/nixpkgs-update/blob/main/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

> [!TIP]
> As a maintainer, if your package is located under `pkgs/by-name/*`, you can comment **`@NixOS/nixpkgs-merge-bot merge`** to automatically merge this update using the [`nixpkgs-merge-bot`](https://github.com/NixOS/nixpkgs/blob/master/ci/README.md#nixpkgs-merge-bot).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

r-ryantm committed:
python3Packages.onecache: 0.7.1 -> 0.8.0

dotlambda commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 461822`
Commit: `18bb533e337d74f2b29167fd5fa949c836b2701a`

---
### `x86_64-linux`
<details>
  <summary>:x: 4 packages failed to build:</summary>
  <ul>
    <li>python312Packages.aiosonic</li>
    <li>python312Packages.aiosonic.dist</li>
    <li>python313Packages.aiosonic</li>
    <li>python313Packages.aiosonic.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 4 packages built:</summary>
  <ul>
    <li>python312Packages.onecache</li>
    <li>python312Packages.onecache.dist</li>
    <li>python313Packages.onecache</li>
    <li>python313Packages.onecache.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>python312Packages.aiosonic</summary>
<pre>unpacking source archive /nix/store/kmm8mi799n8ybp2v53409cdv7zsxrj3l-source
source root is source
setting SOURCE_DATE_EPOCH to timestamp 315619200 of file &quot;source/tox.ini&quot;
Running phase: patchPhase
Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
no configure script, doing nothing
Running phase: buildPhase
Executing pypaBuildPhase
Creating a wheel...
pypa build flags: --no-isolation --outdir dist/ --wheel
* Getting build dependencies for wheel...
* Building wheel...
Successfully built aiosonic-0.26.0-py3-none-any.whl
Finished creating a wheel...
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for aiosonic-0.26.0-py3-none-any.whl
  - onecache&lt;0.8.0,&gt;=0.7.0 not satisfied by version 0.8.0</pre>
</details>
</details>

dotlambda committed:
python3Packages.aiosonic: 0.26.0 -> 0.30.1

Diff: https://github.com/sonic182/aiosonic/compare/0.26.0...0.30.1

Changelog: https://github.com/sonic182/aiosonic/blob/0.30.1/CHANGELOG.md

dotlambda commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 461822`
Commit: `7015cb912f36c905466aa39b429a693edeae7c03`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 8 packages built:</summary>
  <ul>
    <li>python312Packages.aiosonic</li>
    <li>python312Packages.aiosonic.dist</li>
    <li>python312Packages.onecache</li>
    <li>python312Packages.onecache.dist</li>
    <li>python313Packages.aiosonic</li>
    <li>python313Packages.aiosonic.dist</li>
    <li>python313Packages.onecache</li>
    <li>python313Packages.onecache.dist</li>
  </ul>
</details>

geraldog reviewed (approved):
Thanks @dotlambda  !
